### ISSUE TITLE
overlay notebook semantics onto the screen reader with `aria-roledescription`

### ISSUE THREAD

**Issue by tonyfast** [deathbeds/nbconvert-a11y#28]
we use the following generic aria roles `region` and  `group` to describe cells, inputs and outputs. we actually overlay notebook semantics onto the screen reader with [`aria-roledescription`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-roledescription). when this feature is added, it should be easy to toggle on/off. screen reader power users might prefer familiar screen reader cues while renamed descriptions are more conversational.

**Comment by tonyfast** [comment]
marking up notebooks semantic might be a plausible uses case:

>  If you have a very specific audience, that is homogeneous in language, in skill, in experience, and in technology, and that audience has struggled with a specific concept or control, then sure, maybe experiment. 

> There is some guidance: no blank values, only use it on things that already have implicit or explicit roles, and use it only to clarify a thing. 

https://adrianroselli.com/2020/04/avoid-aria-roledescription.html
